core/climate.mcf

```diff
@@ -63,6 +63,18 @@ subClassOf: schema:Place
 name: "GeoGridPlace_1Deg"
 description: "A place representing a uniform 1x1 degree grid on the surface of the Earth. Unlike IPCCPlace entities, these are not defined in the context of a country."
 
+Node: dcid:GeoGridPlace_025Deg
+typeOf: schema:Class
+subClassOf: schema:Place
+name: "GeoGridPlace_025Deg"
+description: "A place representing a uniform 0.25x0.25 degree grid on the surface of the Earth; contained in US Counties."
```

Comment:
nit: We can remove "contained in US Counties" (+below), since entities of this type can be located anywhere else.